charts/kuberpult/values.yaml.tpl

```diff
@@ -25,6 +25,7 @@ cd:
     requests:
       cpu: 1
       memory: 2Gi
+  pvc_storage: 5Gi
```

Comment:
Shouldn't this rather look like this:

```
pvc:
  storage: 5Gi
```

We probably need to support annotations or storage classes eventually.

Reply:
Makes sense, I modified the code, thanks for the hint